examples/chromebook-linux.conf

```diff
@@ -0,0 +1,54 @@
+# NOTE: to use this, rename this file to default.conf or `your-chomebook-keyboard-name`.conf and put in /etc/keyd/
+
+# This is an chromebook like configuration, this remaps function keys 1 through 10 to fall in line with the chromeos remapping
+# while still allowing for full access to standard linux functions that rely on those function keys, such as TTY access or
+# terminal programs
+
+# Using this on an chromebook that has been modified to run linux will make the function keys work as designed originally 
+# for the most part, F4 and F5 are mapped differently due to limitations of `keyd` and lack of functional support for their features
+# in most linux desktop environments
+
+# pressing meta+Fn will trigger that FN key
+# pressing ctrl+alt+Fn will trigger that key stroke
+
+[ids]
+*
+
+## Chromebook function keys
+[main]
+f1 = back
+f2 = forward
+f3 = refresh
+f4 = print
+f5 = A-tab 
+f6 = brightnessdown
+f7 = brightnessup
+f8 = mute
+f9 = volumedown
+f10 = volumeup
+
+## Allow F1-10 access through meta+fnX
+[meta]
+f1 = f1
+f2 = f2
+f3 = f3
+f4 = f4
+f5 = f5
+f6 = f6
+f7 = f7
+f8 = f8
+f9 = f9
+f10 = f10
+
+## Allow TTY access
+[control+alt]
+f1 = C-A-f1
+f2 = C-A-f2
+f3 = C-A-f3
+f4 = C-A-f4
+f5 = C-A-f5
+f6 = C-A-f6
+f7 = C-A-f7
+f8 = C-A-f8
+f9 = C-A-f9
+f10 = C-A-f10
```

Comment:
I have some more suggestions:

```suggestion
f10 = C-A-f10
left = home
right = end

[control+alt]
up = home
down = end

[alt]
backspace = delete
up = pageup
down = pagedown
```

Reply:
Will need to play with these shortly, 
AltBackspace is used by a few programs iirc, hence why I chose MetaBackspace for delete

Reply:
It's documented here: https://support.google.com/chromebook/answer/183101?hl=en

Text editing / Delete the next letter (forward delete): Alt + Backspace
Page & web browser / Page up	: Alt + Up arrow
Page & web browser / Page down	: Alt + Down arrow
Page & web browser / Go to top of page: Ctrl + Alt + Up arrow
Page & web browser / Go to bottom of page: Ctrl + Alt + Down arrow

But feel free to omit alt-backspace, the others are more important to me.